docs/08-developer-guide/04-project/02-release-guide.md

```diff
@@ -24,6 +33,25 @@ brew install gnu-tar
 export PATH="$HOMEBREW_PREFIX/opt/gnu-tar/libexec/gnubin:$PATH"
 ```
 
+You may encounter problems regarding sqlite3 compatibility issues when installing Subversion on Mac. Use the following workaround to resolve it:
+
+```bash title="Install Subversion on Mac"
+# fix svn sqlite3 incompatibility
+brew update
+brew remove sqlite svn
+brew reinstall sqlite svn --build-from-source
+```
+
+</TabItem>
+<TabItem value="linux" label="Linux">
```

Comment:
I think we should somehow indicate the specific Linux distro these `yum` commands were tested on.  Various package managers exist and even package names tend to be different across them.

Reply:
I'll verify with different flavors of Linux.